services/common-features.yaml

```diff
@@ -118,3 +117,13 @@ features:
     description: |
       Provides the ability to copy data or resource to multiple locations to ensure
       availability and durability.
+
+  - id: CCC.F22 # Location Lock-In
```

Comment:
Can you add these to the common features snippets?

Reply:
Added